eng/dashboardpack/dashboardpack.csproj

```diff
@@ -27,16 +27,16 @@
   <Target Name="Build" />
 
   <Target Name="BeforeBuild" BeforeTargets="Build">
-    <MSBuild Projects="../../src/Aspire.Dashboard/Aspire.Dashboard.csproj" Targets="publish" Properties="Configuration=$(Configuration);Platform=$(Platform);TargetFramework=$(TargetFramework);RuntimeIdentifier=$(DashboardRuntime)" />
+    <MSBuild Projects="../../src/Aspire.Dashboard/Aspire.Dashboard.csproj" Targets="publish" Properties="Configuration=$(Configuration);Platform=$(Platform);TargetFramework=$(TargetFramework);RuntimeIdentifier=$(DashboardRuntime);PlatformName=$(DashboardRuntime)" />
```

Comment:
Just curious - do you have a link to why this fixed publishing symbols?

Reply:
Sorry, should have explained better. As you can see here, this MSBuild command is handling the multi-targeting of the project for different RIDs. During the inner builds, we calculate what the Symbol store output dir would be based on various things, including the platform name:

![image](https://github.com/dotnet/aspire/assets/13854455/180fe90a-2d3a-48df-ad4f-ae627d5bb07e)

Before this change, we were not setting PlatformName, which was causing all different innerbuilds to push their symbols to the same location, meaning they would override the previous inner build's symbols, which also meant that in the end we were only pushing the symbols for one platform, as opposed to all:

![image](https://github.com/dotnet/aspire/assets/13854455/84b9dc54-6b93-4633-b88f-0d30b7af8f02)

![image](https://github.com/dotnet/aspire/assets/13854455/43a212df-b836-4821-b4af-cf28e8852dd3)


By setting Platform name correctly on the inner builds, we fix this as each inner build will have its unique symbol store path, and therefore symbols for all platforms are published.

Let me know if that doesn't make sense, and I'm happy to ellaborate more.

Reply:
Thanks for the detailed explanation! IIUC, normally `PlatformName` would be just the arch, in which case that needs to be extracted from `$(DashboardRuntime)`. But since the same build is producing symbols for multiple os+arch for our case, I guess this will be fine too. Hopefully, `PlatformName` to `win-x86` won't affect anything else.

Reply:
I think Platform is the one that is widely used by the rest of the build system. I checked and PlatformName is only used in a handful of places, mainly things around Output locations.